File: 백준/Bronze/6810. ISBN/ISBN.c
#include<stdio.h>
int main(void){
    int a,b,c;
    scanf("%d%d%d",&a,&b,&c);
    int sum = 9 + 7 * 3 + 8 + 0 * 3 + 9 + 2 * 3 + 1 + 4 * 3 + 1 + 8 * 3;
    sum += a + b * 3 + c;
    printf("The 1-3-sum is %d\n",sum); 
    return 0;
}

File: 백준/Bronze/6810. ISBN/ISBN.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int sum = 9 + 7 * 3 + 8 + 0 * 3 + 9 + 2 * 3 + 1 + 4 * 3 + 1 + 8 * 3;
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        sum += a + b * 3 + c;
        System.out.println("The 1-3-sum is " + sum);      
    }
}